framework/src/auxkernels/VariableTimeIntegrationAux.C

```diff
@@ -28,7 +28,9 @@ VariableTimeIntegrationAux::validParams()
 VariableTimeIntegrationAux::VariableTimeIntegrationAux(const InputParameters & parameters)
   : AuxKernel(parameters),
     _coef(getParam<Real>("coefficient")),
-    _order(getParam<unsigned int>("order"))
+    _order(getParam<unsigned int>("order")),
+    _u_old(_order != 3 ? uOld() : genericZeroValue<false>()),
+    _u_older(_order == 3 ? uOlder() : genericZeroValue<false>())
```

Comment:
Hmmm - is there no better way?  It would be great to avoid the ternary.  Maybe it would be better to hide this in the interface underneath `uOld()`?

Reply:
This is specific to this kernel only. Don't need the older soln if the order is < 3.

I could make it a pointer...

What I was more concerned about was the `uOld()` and `uOlder()` interface.

Reply:
Oh - I see what you're saying.  It looks good then.